Salwa's pull request for making Cow class

sms-sudo opened the pull request:
Created new Cow class that implements tradeable and domesticatable
Includes test file
Changed trader to initialize a Cow

sms-sudo committed:
Adding Tradable Cow Class and accompanying CowTests

sms-sudo committed:
Adding Tradable Cow Class and accompanying CowTests

marianmwang commented on src/test/java/CowTest.java:
```diff
@@ -0,0 +1,34 @@
+/*
+ * This file contains sample JUnit test cases for Horse.java
```
this is for Cow.java

sms-sudo replied:
I just forgot to change it... but I'll count it as an intended error. :))

marianmwang commented on src/test/java/CowTest.java:
```diff
@@ -0,0 +1,34 @@
+        assertEquals("I'm a cow and the noise I make is Moo!", c.toString());
+    }
+
+}
```
misisng tests! should include test for your new SetColour method

sms-sudo replied:
Yes !

marianmwang commented on src/main/java/Cow.java:
```diff
@@ -0,0 +1,29 @@
+        this.colour = "brown";
+    }
+
+    public void Setcolour(String colour) {
```
bad naming convention, should be "setColour"

sms-sudo replied:
Yes !

Silk8Worm replied:
^

sms-sudo committed:
Fixed issues:
- type in head doc. string
- fixed naming issue in setter method (Cow class)
- tested setColour (Cow test)

sms-sudo committed:
Merge branch 'master' into sms-sudo

sms-sudo committed:
Merge branch 'master' into sms-sudo
